boards/arm/nrf9160_pca10090/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# Kconfig - nRF91 PCA10090 board configuration
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF9160_PCA10090
+
+config BOARD
+	default "nrf9160_pca10090"
+

```

Comment:
Please add here:
```
if ADC

config ADC_0
	default y

endif # ADC

if I2C

config I2C_0
	default y

endif # I2C

if SPI

config SPI_1
	default y

endif # SPI
```
to enable default instances, like it is done for other Nordic DK boards.
This will allow users to build samples and tests that use these peripherals without extra configuration.

Reply:
Thanks. Will do

Reply:
Done